lib/i18n_routable/translation_assistant.rb

```diff
@@ -15,16 +15,18 @@ def untokenize_segment segment
 
     # /posts => ["/:i18n_posts", ["posts"]]
     def convert_path_to_localized_regexp path, requirements={}, anchor= true
-      spec = Journey::Parser.new.parse(path)
+      parser = ActionDispatch::Journey::Parser.new
+      spec = parser.parse(path)
       visitor = RouteTranslationVisitor.new
       new_path = visitor.accept(spec)
+      new_path_spec = parser.parse(new_path)
       new_requirements = requirements.merge(visitor.route_translations)
-      [new_path, new_requirements]
+      [new_path, new_path_spec, new_requirements]
     end
 
     def translate_segment segment, locale
       segment = untokenize_segment segment
-      CGI.escape I18n.t segment, default: segment, scope: 'routes', locale: locale
+      I18n.t segment, default: segment, scope: 'routes', locale: locale
```

Comment:
The updated Journey appears to already handle the escaping. So with the CGI.escape, it was double-escaping, which resulted in
```
expected: "/es/caf%C3%A9"
            got: "/es/caf%25C3%25A9"
```
[localizable_formatter:44](https://github.com/change/i18n_routable/blob/master/lib/i18n_routable/outgoing/localizable_formatter.rb#L44) calls `route.format`
[route.rb:68](https://github.com/rails/rails/blob/3e36db4406beea32772b1db1e9a16cc1e8aea14c/actionpack/lib/action_dispatch/journey/route.rb#L68) calls `@path_formatter.evaluate`
[visitors.rb:43](https://github.com/rails/rails/blob/3e36db4406beea32772b1db1e9a16cc1e8aea14c/actionpack/lib/action_dispatch/journey/visitors.rb#L43) which calls `param.escape`.